src/DepressurizerCore/Models/DatabaseEntry.cs

```diff
@@ -780,8 +781,11 @@ private AppType ScrapeStoreHelper(int id)
                 result = AppType.Unknown;
             }
 
-            LastStoreScrape = DateTimeOffset.Now.ToUnixTimeSeconds();
-
+            //Save last AppInfoUpdate
+            if (result.Equals(AppType.Game) || result.Equals(AppType.Application))
+            {
+                LastAppInfoUpdate = DateTimeOffset.UtcNow.ToUnixTimeSeconds();
```

Comment:
LastAppInfoUpdate refers to the appinfo.vdf file in your Steam folder. This has nothing to do with scraping.

Reply:
The value public long LastAppInfoUpdate { get; set; } = 0; in  DatabaseEntry is as I see the value later save in the json file "LastAppInfoUpdate": 0,`. When it not set different to 0. Their are the the side effect:
*Games loose their category when LastStoreScrape not set.